chromadb/test/property/strategies.py

```diff
@@ -69,24 +69,18 @@ def one_or_both(strategy_a, strategy_b):
     )
 
 
+# Temporarily generate only these to avoid SQL formatting issues.
+legal_characters = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789-_./+"
+
+
 @st.composite
 def unique_ids_strategy(draw, count: int):
 
-    ratio = 20
-    strs = count // ratio
-
-    str_results = draw(
-        st.lists(st.text(min_size=1, max_size=64), min_size=strs, max_size=strs, unique=True)
-    )
+    strat = st.text(alphabet=legal_characters, min_size=1, max_size=64)
```

Comment:
Is there a reason we got rid of uuid gen too? assuming its just because this covers uuid range as well but wanted to confirm.

Reply:
UUID gen was a performance improvement to get more predictable behavior. After a few other (seemingly unrelated) changes, text() seems to be working fine again and we're getting reasonable perf and distribution of tests.

This is one thing I don't like about Hypothesis... it's subject to certain kinds of "spooky action at a distance" changes. Let's discuss in the office this week.